debian/linuxcnc.install.in

```diff
@@ -1 +1,261 @@
+usr/share/man/man1/halrun.1
+usr/share/man/man1/halsampler.1
+usr/share/man/man1/halscope.1
+usr/share/man/man1/halshow.1
+usr/share/man/man1/halstreamer.1
+usr/share/man/man1/haltcl.1
+usr/share/man/man1/halui.1
+usr/share/man/man1/hbmgui.1
+usr/share/man/man1/hexagui.1
+usr/share/man/man1/hy_gt_vfd.1
+usr/share/man/man1/hy_vfd.1
+usr/share/man/man1/image-to-gcode.1
+usr/share/man/man1/inivar.1
+usr/share/man/man1/io.1
+usr/share/man/man1/iocontrol.1
+usr/share/man/man1/iov2.1
+usr/share/man/man1/latency-histogram.1
+usr/share/man/man1/latency-plot.1
+usr/share/man/man1/latency-test.1
+usr/share/man/man1/lineardelta.1
+usr/share/man/man1/linuxcnc.1
+usr/share/man/man1/linuxcnc_info.1
+usr/share/man/man1/linuxcnclcd.1
+usr/share/man/man1/linuxcncmkdesktop.1
+usr/share/man/man1/linuxcnc_module_helper.1
+usr/share/man/man1/linuxcncrsh.1
+usr/share/man/man1/linuxcncsvr.1
+usr/share/man/man1/linuxcnctop.1
+usr/share/man/man1/linuxcnc_var.1
+usr/share/man/man1/maho600gui.1
+usr/share/man/man1/max5gui.1
+usr/share/man/man1/mb2hal.1
+usr/share/man/man1/mdi.1
+usr/share/man/man1/mdro.1
+usr/share/man/man1/milltask.1
+usr/share/man/man1/mitsub_vfd.1
+usr/share/man/man1/monitor-xhc-hb04.1
+usr/share/man/man1/motion-logger.1
+usr/share/man/man1/moveoff_gui.1
+usr/share/man/man1/ngcgui.1
+usr/share/man/man1/panelui.1
+usr/share/man/man1/pi500_vfd.1
+usr/share/man/man1/pmx485.1
+usr/share/man/man1/pmx485-test.1
+usr/share/man/man1/pncconf.1
+usr/share/man/man1/puma560gui.1
+usr/share/man/man1/pumagui.1
+usr/share/man/man1/pyngcgui.1
+usr/share/man/man1/pyui.1
+usr/share/man/man1/pyvcp.1
+usr/share/man/man1/pyvcp_demo.1
+usr/share/man/man1/qtplasmac-cfg2prefs.1
+usr/share/man/man1/qtplasmac-materials.1
+usr/share/man/man1/qtplasmac-plasmac2qt.1
+usr/share/man/man1/qtplasmac-setup.1
+usr/share/man/man1/qtvcp.1
+usr/share/man/man1/rotarydelta.1
+usr/share/man/man1/rs274.1
+usr/share/man/man1/rtapi_app.1
+usr/share/man/man1/scaragui.1
+usr/share/man/man1/schedrmt.1
+usr/share/man/man1/scorbot-er-3.1
+usr/share/man/man1/sendkeys.1
+usr/share/man/man1/shuttle.1
+usr/share/man/man1/sim_pin.1
+usr/share/man/man1/sim-torch.1
+usr/share/man/man1/simulate_probe.1
+usr/share/man/man1/stepconf.1
+usr/share/man/man1/svd-ps_vfd.1
+usr/share/man/man1/teach-in.1
+usr/share/man/man1/thermistor.1
+usr/share/man/man1/tooledit.1
+usr/share/man/man1/touchy.1
+usr/share/man/man1/update_ini.1
+usr/share/man/man1/vfdb_vfd.1
+usr/share/man/man1/vfs11_vfd.1
+usr/share/man/man1/wj200_vfd.1
+usr/share/man/man1/xhc-hb04.1
+usr/share/man/man1/xhc-hb04-accels.1
+usr/share/man/man1/xhc-whb04b-6.1
+usr/share/man/man1/xyzac-trt-gui.1
+usr/share/man/man1/xyzbc-trt-gui.1
+
+usr/share/man/man9
+
+usr/share/qtvcp
+
+src/emc/usr_intf/axis/etc    usr/share/doc/@MAIN_PACKAGE_NAME@/axis
+src/emc/usr_intf/axis/README usr/share/doc/@MAIN_PACKAGE_NAME@/axis
+
 debian/linuxcnc-uspace.metainfo.xml usr/share/metainfo
+
+debian/extras/etc /
+debian/extras/lib /
+
+debian/extras/usr/share/icons usr/share
```

Comment:
Would it be possible to specify `linuxcncicon.svg` (and `linux-logo.svg` , not sure if this is used anywhere). Then you could include the other icons in their respective documentation packages. That would solve #1655